app/models/processing_destinations_lookup.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class ProcessingDestinationsLookup < ApplicationRecord
```

Comment:
I feel the name `ProcessingDestinationsLookup` is unnecessarily long. `ProcessingDestination` is more than enough

Reply:
also it would kinda mean that the db table is a table of lookups
whereas it _is_ a lookup table, of processing destinations